ROADMAP.md

```diff
@@ -59,17 +47,55 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
     * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
 * Minio
   * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+
+
+## Priority future work
+
+* Custom resource validation, progress, status [#1539](https://github.com/rook/rook/issues/1539)
+* Integration testing improvements
+  * Update promotion and release channels to align with storage provider specific statuses [#1885](https://github.com/rook/rook/issues/1885)
+  * Isolate and parallelize storage provider testing [#1218](https://github.com/rook/rook/issues/1218)
+  * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
+  * Include more architectures in test pipeline [#1901](https://github.com/rook/rook/issues/1901)
+  * Include more environments in test pipeline [#1315](https://github.com/rook/rook/issues/1315)
+* Support for dynamic provisioning of new storage types
+  * Dynamic bucket provisioning [#1705](https://github.com/rook/rook/issues/1705)
+  * Dynamic database provisioning [1704](https://github.com/rook/rook/issues/1704)
+* Cassandra
+  * Continue to implement Cassandra design [2294](https://github.com/rook/rook/issues/2294)
+* Ceph
+  * Support modified CRUSH maps [#2514](https://github.com/rook/rook/issues/2514)
+  * Config
+    * Use `/etc/ceph/` and `/var/lib/ceph` default locations inside containers [#1368](https://github.com/rook/rook/issues/1368)
+    * Update configs for latest best-practices [#1347](https://github.com/rook/rook/issues/1347)
+  * Mons
+    * Revise mon startup/update procedures for most recent best-practices [#2321](https://github.com/rook/rook/issues/2321)
+    * Mon placement respects failure domains [#2603](https://github.com/rook/rook/issues/2603)
+  * OSDs
+    * Run on arbitrary PVs (local storage) as an alternative to host path [#919](https://github.com/rook/rook/issues/919)
+    * Allow multiple osds on fast device types (NVMe) [#2640](https://github.com/rook/rook/issues/2640)
```

Comment:
This is already implemented, we can remove this. This issue is a more specific improvement.